src/Access/Common/AccessRightsElement.h

```diff
@@ -11,12 +11,17 @@ namespace DB
 struct AccessRightsElement
 {
     AccessFlags access_flags;
+
     String database;
     String table;
     Strings columns;
+    String parameter;
+
     bool any_database = true;
     bool any_table = true;
     bool any_column = true;
+    bool any_global_with_parameter = true;
```

Comment:
`any_global_with_parameter` is hard to read. Couldn't it be just `any_parameter`?

Reply:
done